Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -207,8 +207,9 @@ public void OnSetUp ([NotNull] string testName)
     /// </summary>
     /// <param name="maximizeWindow">Specified whether the new browser session's window should be maximized.</param>
     /// <param name="configurationOverride">Specifies additional options applied when creating the browser.</param>
+    /// <param name="headlessSize">If not set, a value of 1050x868 is set as the browser size for headless windows.</param>
```

Comment:
value doesn't match the size in line 225. 

The size used in line 225 is specifically for our tests, correct? then we should set it there. 1296x868 does not match a standard screen size such as 1280x1024 because it's wider than 1280. 

Looking further, the maximizeWindow paramerter and the headlessSize parameter are mutually exclusive, aren't they? Not sure how to best handle this, given we have the DriverConfigOverride which controls the headlessmode.

I also noticed that the headless tests still have a maxmizeWindow flag. This is no longer sensible with headless, isn't it? I think a bit more cleanup would be a good thing for splitting the code into UI and headless. We should discuss what is needed and if it should be in here or a separate task.

Reply:
Do we want the size to actually match a reasonable screensize? If yes, I would create a ticket for that.

Reply:
need to read this comment.

Reply:
This means, to have a good default for headless instead of something the project developer must configure?

Ah, idea! and, yes, can be separate ticket if you want:

in WebTestHelper.CreateNewBrowserSession, we already use maximize if the size is null. We could simply use a fixed size as the null fallback value if it's headless:
line 220: isntead of `windowSize ??= WindowSize.Maximized;` we use `windowSize ??= configuration.Headless ? new WindowSize(1280, 900): WindowSize.Maximize`